Source/Evolve/WarpXEvolve.cpp

```diff
@@ -385,7 +385,6 @@ WarpX::OneStep_nosub (Real cur_time)
         EvolveF(0.5*dt[0], DtType::SecondHalf);
 
         EvolveB(0.5*dt[0]); // We now have B^{n+1}
-
```

Comment:
i would add the new-line back in so this doesn't have to propagate all the way to WarpX master.